MainApp/MainApp/Feature/Auth/Register/RegisterViewController.swift

```diff
@@ -111,8 +111,10 @@ class RegisterViewController: MasterViewController, RegisterViewProtocol {
                           tfConfirmPassword]
         for textfield in textFields {
             textfield?.addTarget(self, action: #selector(handleField), for: .editingDidEnd)
+            textfield?.addTarget(nil, action: Selector(("firstResponderAction:")), for: .editingDidEndOnExit)
```

Comment:
is this still used

Reply:
masih, untuk hide keyboard pas klik return